content/docs/0.6.0/reference/api/index.md

```diff
@@ -9,8 +9,9 @@ In this section, the functionality and commands of the Keptn REST API are descri
 
 ## Prerequisites
 
-- To access the Keptn API, a running Keptn installation is needed. If you have not set up Keptn yet, [please start here](../../installation/setup-keptn).
+- To access the Keptn API, a running Keptn installation is needed. If you have not set up Keptn yet, please start [here](../../installation/setup-keptn).
 
+- To get the **API token** for the basic authentication of API calls, please see [here](../cli/#keptn-auth).  
```

Comment:
Actually, the API token provides no "basic authentication".

Reply:
Valid point. Fixed the wording.

Reply:
Thx!